R/teal_data-class.R

```diff
@@ -0,0 +1,88 @@
+setOldClass("JoinKeys")
+
+#' Reproducible data.
+#'
+#' Reproducible data container class. Inherits code tracking behavior from [`teal.code::qenv-class`].
+#'
+#' This class provides an isolated environment in which to store and process data with all code being recorded.
+#' The environment, code, data set names, and data joining keys are stored in their respective slots.
+#' These slots should never be accessed directly, use the provided get/set functions.
+#'
+#' As code is evaluated in `teal_data`, messages and warnings are stored in their respective slots.
+#' If errors are raised, a `qenv.error` object is returned.
+#'
+#' @name teal_data-class
+#' @rdname teal_data-class
+#'
+#' @slot env (`environment`) environment containing data sets and possibly auxiliary variables
+#'  Access variables with [get_var()] or [`[[`].
```

Comment:
```suggestion
#'  Access variables with [`get_var`] or [`[[`].
```

Reply:
what is wrong with `[get_var()]`

Reply:
It's a function call, usually docs mention function _names_, and in `code` format.